src/Method/GetSlots.php

```diff
@@ -50,18 +51,24 @@ protected function constructResponseData($data)
 
 		$result = array();
 
-		foreach ($data as $index => $slot) {
+		foreach ($responseData as $index => $slot) {
 			$whichSlot = sprintf('(slot #%d)', $index + 1);
 			$startDateTime = $slot['startDateTime']->format(DateTime::W3C);
 			$endDateTime = $slot['endDateTime']->format(DateTime::W3C);
+			$startDate = $slot['startDateTime']->format('Y-m-d');
+			$requestedDate = $requestData[0]->format('Y-m-d');
 			$capacity = $slot['capacity'];
 
 			if ($startDateTime >= $endDateTime) {
 				throw new ValidationException(sprintf('Slot startDateTime (%s) must be before endDateTime (%s) %s', $startDateTime, $endDateTime, $whichSlot));
 			}
 
+			if ($requestedDate !== $startDate) {
```

Comment:
[Je to tam](https://github.com/helecz/php-partner-sync-api/blob/master/readme.md#ongetslots) v podstatě